src/bc.F

```diff
@@ -2956,6 +2956,8 @@ subroutine SetGPUParams(openacc,device)
          logical, intent(inout) :: openacc
          logical, optional :: device
 
+      ! Set logical flag if subroutine is called on GPU or CPU resident
+      ! data. The default to for GPU resident data.
```

Comment:
What does `the default to for GPU resident data` mean?

Reply:
I corrected the comment.  Please verify that it makes more sense now.

Reply:
Thanks John. It looks good to me now.